### PR TITLE
White list tables

### PR DESCRIPTION
**Pull request by bigbadtrumpet**

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (altendo) [inline .idea/codeStyleSettings.xml:1]: `.idea` files should probably be added in a separate commit, if at all. Ideally editor-specific formatting rules should be ignored by git (ie there should be a .gitignore rule for these files).

File: package.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
     "lint": "eslint src",
     "check": "flow check",
     "build": "gulp dist",
-    "start": "node ./scripts/start.js",
+    "start": "gulp dist && node ./scripts/start.js",
```
Comment (altendo) [inline package.json:23]: rather than redefine `start` and create a `start-norebuild` option, leave it alone and create a `start-rebuild` option. That will maintain `start`'s functionality and offer the new functionality without breaking existing applications.

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (bigbadtrumpet) [reply .idea/codeStyleSettings.xml:1]: IMO we should share these files, so if someone uses the same editor, the settings that should be shared are, which is why i added them :)

File: package.json (hunk 2 above)
Comment (bigbadtrumpet) [reply package.json:23]: Yea the only reason I changed this, was because i assumed that running start would actually update the code and then run it, and i didnt want anyone else to make this mistake.  But im cool with changing it back

File: src/Provisioner.js [hunk 3]
```diff
@@ -6,8 +6,8 @@ import Throughput from './utils/Throughput';
 import ProvisionerLogging from './provisioning/ProvisionerLogging';
 import { Region } from './configuration/Region';
 import DefaultProvisioner from './configuration/DefaultProvisioner';
-import { invariant } from './Global';
-import type { TableProvisionedAndConsumedThroughput, ProvisionerConfig, AdjustmentContext } from './flow/FlowTypes';
+import { invariant, log } from './Global';
+import type { TableProvisionedAndConsumedThroughput, ProvisionerConfig, AdjustmentContext} from './flow/FlowTypes';
```
Comment (altendo) [inline src/Provisioner.js:10]: probably should leave the whitespace next to AdjustmentContext alone

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (yushun) [reply .idea/codeStyleSettings.xml:1]: It depends really, If the styles is enforced to all to follow it then yes.
But it maybe inconvenient if everyone is changing the code commit to base on their style.

On way people solve this is to have a separate repo to keep styles or IDE specific things

**Comment by altendo** [comment]
closing, already merged the changes
